docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/FieldAwareFactorizationMachineWithOptions.tt

```diff
@@ -6,17 +6,23 @@ string Trainer = "FieldAwareFactorizationMachine";
 string OptionsInclude = @"using Microsoft.ML.Trainers;";
 
 string Comments = @"
-        // This example first train a field-aware factorization to binary classification, measure the trained model's quality, and finally
+        // This example first train a field-aware factorization to binary
+        // classification, measure the trained model's quality, and finally
         // use the trained model to make prediction.";
 
-string TrainerDescription = @"// This trainer trains field-aware factorization (FFM) for binary classification. See https://www.csie.ntu.edu.tw/~cjlin/papers/ffm.pdf
-            // for the theory behind and https://github.com/wschin/fast-ffm/blob/master/fast-ffm.pdf for the training
-            // algorithm implemented in ML.NET.";
+string TrainerDescription = @"// This trainer trains field-aware factorization (FFM)
+            // for binary classification.
+            // See https://www.csie.ntu.edu.tw/~cjlin/papers/ffm.pdf for the theory
+            // behind and
+            // https://github.com/wschin/fast-ffm/blob/master/fast-ffm.pdf for the
+            // training algorithm implemented in ML.NET.";
 
 string TrainerOptions = @"FieldAwareFactorizationMachineTrainer.Options
             {
                 FeatureColumnName = nameof(DataPoint.Field0),
-                ExtraFeatureColumns = new[] { nameof(DataPoint.Field1), nameof(DataPoint.Field2) },
+                ExtraFeatureColumns = 
+            new[] { nameof(DataPoint.Field1), nameof(DataPoint.Field2) },
+
```

Comment:
Is the indentation in line 24 of this tt file different than the norms we set because of spacing issues in the cs file?